atlasdb-client/build.gradle

```diff
@@ -34,11 +34,11 @@ dependencies {
   compile(group: "com.google.protobuf",
           name: "protobuf-java",
           version: "2.6.0")
-  compile('com.netflix.feign:feign-jackson:8.6.1') {
+  compile('com.netflix.feign:feign-jackson:8.17.0') {
```

Comment:
Could we extract this into a variable, maybe `libVersions.feign`? (I think similar ones live in `versions.groovy`)